static/app/views/dashboards/widgetBuilder/components/widgetBuilderSlideout.tsx

```diff
@@ -95,7 +98,14 @@ function WidgetBuilderSlideout({
           borderless
           aria-label={t('Close Widget Builder')}
           icon={<IconClose size="sm" />}
-          onClick={onClose}
+          onClick={() => {
+            openConfirmModal({
+              bypass: isEqual(initialState, state),
+              message: t('You have unsaved changes. Are you sure you want to leave?'),
+              priority: 'danger',
+              onConfirm: () => onClose(),
```

Comment:
```suggestion
              onConfirm: onClose,
```

In cases where all we're doing is calling the function and the props of both the callsite and the function are the same, I just shorthand like this it so we don't need to create another anonymous function